platform/administer/platform-database/database.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Platform Database
+sidebar_label: Database
+sidebar_position: 1
+description: Information that an admin may need to know about the platform database.
+---
+
+import BasePrerequisites from '../../_partials/install/base-prerequisites.mdx';
+
+import InstallNextSteps from '../../_partials/install/install-next-steps.mdx';
+
+import VersionBadge from '@site/src/components/VersionBadge';
+
+<VersionBadge platformVersion="v4.4.0"/>
+
+The platform database is a local postgres database server that is used and managed by vCluster Platform.
+
+## Configure
+
+By default, platform database will leverage the default StorageClass. If you would like to specify a different StorageClass, you can navigate to `Admin > Config` and edit the  `platformDB.storageClass` field.
+
+### Troubleshoot
+
+The PersistentVolumeClaim. that backs the platform database should never be deleted. It is named `platform-db`. In the case that it is deleted, the following steps can be followed to return the platform database to a working state.
```

Comment:
```suggestion
The `PersistentVolumeClaim`, named `platform-db`, that backs the platform database should never be deleted.  In the case that it is deleted, use these steps to recover.
```

Reply:
Does PersistentVolumeClaim need tick marks? I haven't used them for StatefulSet or Deployment in this same doc.